### ISSUE TITLE
ILM documenation update 

### ISSUE THREAD

**Issue by jakelandis** [elastic/beats#11347]
Index Lifecycle Management (ILM) documentation needs to be updated.

For example:
https://www.elastic.co/guide/en/beats/filebeat/7.x/ilm.html
https://www.elastic.co/guide/en/beats/metricbeat/7.x/ilm.html

In 7.x ILM is automatic, no setup required. The documentation should reflect this and describe the default policy. 

In 6.7 and 7.x ILM is no longer consider BETA (at least from Elasticsearch's perspective).

**Comment by jakelandis** [comment]
related: https://github.com/elastic/logstash/issues/10574

**Comment by urso** [comment]
Checking the docs in master and 7.0 I noticed we didn't update them yet.
Some notable changes between 6.7 and 7.0:

Settings have been moved from `output.elasticsearch.index` to `setup.ilm`. For reference see: https://github.com/elastic/beats/blob/master/libbeat/_meta/config.reference.yml#L908

- setup.ilm.enabled supports the values: 'auto', 'false', 'true. If auto is set, ILM will be enabled if: ES 7.0 is used, ES is at least Basic license, ILM is enabled in ES.

Other settings:
- `setup.ilm.policy_name`: Defaults to `<beatname>-<version>`. Confiure an alternative ILM policy name.
- `setup.ilm.policy_file`: (Default empty). Path to JSON file with ILM policy configuration.
- `setup.ilm.check_exists`: (default true). check_exists can disable the check for an existing policy. The check requires read_ilm privileges.  If check is disabled the policy will only be installed if Overwrite is enabled.
- `setup.ilm.overwrite`: (default false) always overwrite policy.
